tests/Panel/PanelTest.php

```diff
@@ -771,85 +794,62 @@ public function testPartialWithComponentComparison(): void
             'b' => 'B'
         ];
 
-        $result = Panel::partial($this->app, 'k-test-view', $data);
-
-        $this->assertSame(['a' => 'A'], $result);
-
-        // via headers
-        $this->app = $this->app->clone([
-            'request' => [
-                'headers' => [
-                    'X-Fiber-Include' => 'a',
-                    'X-Fiber-Component' => 'k-test-view'
```

Comment:
Is it intentional that we aren't testing for headers anymore?

Reply:
I thought about it some more while working on the clean up and I cannot see a single point where this is really needed in our setup. 

I only see two use cases for partial reloads

1) to refresh global data
2) to refresh parts of the current view

The $reload method reload is already perfect for both cases and prevents errors because you can only request the same view again. 

The $go method should not allow requesting partial content at all. It simply doesn't make sense. But instead of adding this weird check in the backend, we could simply fix this in the client implementation by throwing out the only option in $go. 

This has the added benefit of simplifying the backend calls – not just for views, but also for dialogs. In dialogs there's no need for all of this.

Reply:
ok 👍 only concern (but I cannot think it through at the moment) would be partial reloads that still alter the current URL (e.g. partial reload of a section because pagination changed and thus URL with page parameter needs to change too)

Reply:
I'd suggest to look into it again if that should ever get us in trouble.